gdpr_cookies.module

```diff
@@ -1,14 +1,18 @@
 <?php
 require_once('src/Entity/ThirdPartyServiceEntityInterface.php');
 
+function gdpr_cookies_preprocess_page(&$variables) {
+  backdrop_add_library('gdpr_cookies', 'tarteaucitron', TRUE);
```

Comment:
Is it really necessary to add the library twice, in hook_init and in hook_preprocess_HOOK?

One of them should be sufficient, no?

Maybe, for the sake of transparency, just stick with hook_init for now, and consider moving the library attachment in a follow-up, after the security update is out?

Reply:
No, I don't think it is.  `hook_library_info()` wasn't implemented properly (was using `hook_library()`) and adding it twice was one of my early attempts to fix.  I'll test without and remove.